internal/pkg/secret/encryption_other.go

```diff
@@ -38,8 +37,7 @@ func DeriveEncryptionKey(salt []byte) ([]byte, error) {
 		return []byte{}, err
 	}
 	userId := strconv.Itoa(os.Getuid())
-	encryptionKey := pbkdf2.Key([]byte(machineId+userId), salt, iterationNumber, keyLength, sha256.New)
-	return encryptionKey, nil
+	return pbkdf2.Key([]byte(machineId+userId), salt, iterationNumber, keyLength, sha256.New), nil
```

Comment:
Can you break the first argument out, and use `fmt.Sprintf()` to create it? We won't have to use `strconv.Itoa()` that way.